src/commands/buildCommands/configs/rules.js

```diff
@@ -16,8 +16,7 @@ module.exports = {
 				options: {
 					importLoaders: 2,
 					modules: true,
-					localIdentName: '_[name]_[local]__[hash:base64:5]',
-					minimize: true,
```

Comment:
`css-loader` was upgraded to `v1.0.0` in #71. This version doesn't support the `minimize` option and the [CHANGELOG](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#breaking-changes) recommends using `cssnano` with `postcss-loader`.